testing/middleware/builder/builder.go

```diff
@@ -334,6 +345,11 @@ func (p *Builder) handleHeaderRequest(w http.ResponseWriter, req *http.Request)
 	}
 	ax := types.Slot(slot)
 	currEpoch := types.Epoch(ax / params.BeaconConfig().SlotsPerEpoch)
+	//if currEpoch >= params.BeaconConfig().ElectraForkEpoch {
```

Comment:
electra isn't supported yet

Reply:
OK, but we can comment only this `if` and the rest can be uncommented